pkg/gardenlet/controller/backupentry/backupentry/reconciler.go

```diff
@@ -183,14 +189,20 @@ func (r *Reconciler) reconcileBackupEntry(
 		BackupBucketProviderStatus: backupBucket.Status.ProviderStatus,
 	}
 
+	secretLastUpdateTime, err := time.Parse(time.RFC3339, extensionSecret.Annotations[v1beta1constants.GardenerTimestamp])
```

Comment:
If we keep `mustReconcileExtensionSecret`:
What happens for existing clusters where the timestamp is not set yet? I think we have to consider this case as well.